tests/component_tests.py

```diff
@@ -18,6 +21,23 @@ def test_GIVEN_components_directory_THEN_it_contains_the_base_component(self):
         self.assertIn(ComponentUtils.BASE_COMPONENT, self.component_utils.get_configurations_as_list(),
                       "Base component was missing (should be called {})".format(ComponentUtils.BASE_COMPONENT))
 
+    def test_GIVEN_an_instrument_THEN_all_block_pvs_are_interesting(self):
```

Comment:
I wrote it like this since it says that class has tests that are ran once per instrument for all components/configuration. I am not sure how I would refactor it so the framework would know to run it correctly. Maybe having an abstract class would work, and then component test class and configuration would inherit from it, but I am not sure if that works in the context of the test framework and the way the 2 classes are written.

Reply:
There are two ways that I can see to refactor this:
* The inheritance mechanism that you suggested. This will work and is how you would do it in java etc. but it's not overly pythonic
* Create a method, probably in a new file like `tests/common_utils`, that both tests will call